### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lucacioria**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
-a slack bot that helps your team keep commitments
 
-# how to use
+A slack bot that helps your team keep commitments
+
+# quick start
+
+## install
+
+- close this repo, then `npm run build`, `npm run start`
```
Comment (gabro) [inline README.md]: Typo: cloNe

Comment (gabro) [reply README.md]: Also, I think we should "distribute" this a docker image

**Review by gabro — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
 
-# how to use
+A slack bot that helps your team keep commitments
+
+# quick start
+
+## install
+
+- clone this repo, then `npm run build`, `npm run start`
+- with your browser, go to `http://my.pledge.deploy.com/add-to-slack` and authorize pledge on your slack team
```
Comment (gabro) [inline README.md]: I stumbled upon the URL for a few seconds wondering what magical service `http://deploy.com/` was.

Can you change this to `http://pledge.example.com/add-to-slack`?

**Review by gabro — CHANGES_REQUESTED** [review]

**Comment by lucacioria** [comment]
should be ok now, can I merge?

**Pull request merged**